lib/models/ArpEntry.dart

```diff
@@ -1,11 +1,15 @@
+import 'dart:developer';
 import 'dart:io';
 
+import 'package:snifferapp/services/DatabaseService.dart';
+
 /// This class holds the necessary information for an ARP entry
 /// At its core, an ARP entry must contain an IP address and its
 /// corresponding MAC address
 class ArpEntry {
   final String macAddress;
   final InternetAddress ip;
+  var _vendor;
```

Comment:
I guess other way to do it is to make a new class that extends ArpEntry but otherwise
this should be `String` since we know its type